Integrations/src/main/java/io/deephaven/integrations/learn/Gatherer.java

```diff
@@ -1,30 +1,55 @@
 package io.deephaven.integrations.learn;
 
 import io.deephaven.db.v2.sources.ColumnSource;
+// REMOVE THIS AFTER REMOVING makeIndexSet!
+import io.deephaven.db.tables.Table;
 
+/**
+ * Gatherer takes Deephaven columnar data and places it into a buffer to be used by Python.  The Python object will take
+ * data from the buffer and use it to construct a 2d array of specified size.
+ */
 public class Gatherer {
 
+    /**
+     * Generate an index set from a table.
+     * THIS IS JUST FOR EASY PYTHON TESTING.  REMOVE BEFORE PROD.
+     *
+     * @param t table
+     * @return index set
+     */
+    public static IndexSet makeIndexSet(final Table t) {
+        final IndexSet is = new IndexSet(t.intSize());
+
+        for(long idx : t.getIndex()) {
+            is.add(idx);
+        }
+
+        return is;
+    }
```

Comment:
The comment here makes this sound like something critical for testing that is just in the wrong permanent location.  Does this need to be in some unit test related functionality?

Reply:
I just looked at all of the changed files.  There are no unit tests.  That seems related to this note.  We should have 100% test coverage of the Java code that has been added.  Then we should target reasonably high coverage (>50%) for the python.